bot/exts/evergreen/connect_four.py

```diff
@@ -0,0 +1,466 @@
+import asyncio
+import random
+import typing
+from functools import partial
+
+import discord
+import emojis
+from discord.ext import commands
+
+NUMBERS = [
+    ":one:",
+    ":two:",
+    ":three:",
+    ":four:",
+    ":five:",
+    ":six:",
+    ":seven:",
+    ":eight:",
+    ":nine:"
+]
+UNICODE_NUMBERS = [
+    "\u0031\u20e3",
+    "\u0032\u20e3",
+    "\u0033\u20e3",
+    "\u0034\u20e3",
+    "\u0035\u20e3",
+    "\u0036\u20e3",
+    "\u0037\u20e3",
+    "\u0038\u20e3",
+    "\u0039\u20e3",
+]
+CROSS_EMOJI = "\u274e"
+HAND_RAISED_EMOJI = "\U0001f64b"
+Coordinate = typing.Optional[typing.Tuple[int, int]]
+EMOJI_CHECK = typing.Union[discord.Emoji, str]
```

Comment:
What was the reasoning in making this custom check, instead of using `discord.Emoji`?

Reply:
The `discord.Emoji` check doesn’t support the discord default emojis. Hence I had to make a custom check for it.